app/views/include/main.scala.html

```diff
@@ -155,6 +155,6 @@
     insideHeader,
     play.twirl.api.HtmlFormat.empty,
     play.twirl.api.HtmlFormat.empty,
-    Some(uiLayouts.footer_links()),
+    Some(uiLayouts.footer_links(accessibilityFooterUrl=Some(controllers.accessibility.routes.AccessibilityStatementController.get().url))),
```

Comment:
@matthewharris1 

`if(<thetoggle>) {Some(uiLayouts.footer_links(accessibilityFooterUrl=Some(<theurl>))} else {Some(uiLayouts.footer_links())}`

Reply:
Remove the <> as just placeholders :-)